core/src/main/java/jenkins/security/apitoken/TokenUuidAndPlainValue.java

```diff
@@ -29,7 +29,7 @@
 /**
  * Contains information about the token and the secret value.
  * It should not be stored as is, but just displayed once to the user and then forget about it.
- * @since TODO
+ * @since This was added in 2.260 but marked NoExternalUse, opened as Beta in 2.265
```

Comment:
It was unclear to me which of the version numbers is relevant here, so went into more detail for clarity.